include/ndarray.hpp

```diff
@@ -436,7 +477,12 @@ NDARRAY_INLINE const T& NDArray<T>::operator[](size_t i) const {
 }
 
 template <class T>
-NDARRAY_INLINE std::vector<size_t> NDArray<T>::shape() const {
+NDARRAY_INLINE const std::vector<T>& NDArray<T>::data() const {
```

Comment:
This is just a reminder that the name of the method will need to be changed here as well.

Reply:
Thanks for the kind reminder.